Python-path-traversal-files-main/DannyWuzhiheng/Web/IP.py

```
@@ -0,0 +1,303 @@
+        username = request.form['username']  
+        password = request.form['password'] 
+        with open(r'./login.txt','r') as f:
+            a=f.read()
+            a=a.split('\n')
+            for i in a:
+                usps=i.split(' ')
+                if usps[0]==username and usps[1]==password:
+                    resp = make_response(redirect('http://116.62.60.158/index', code=302))
+                    resp.set_cookie("login",username)
+                    #此处没有使用高级方法，以后修复
+                    if username == YOUR_ADMIN:
```

Ruff (0.13.3)

Undefined name `YOUR_ADMIN`

(F821)